### PR TITLE
508 compliance for ident/admin/index.php

### PR DESCRIPTION
**Pull request by NikitaSalikov**
## Before Changes:
Screen Shot 2023-09-07 at 11 44 08 AM

### Condensed mode: 
Screen Shot 2023-09-07 at 11 42 52 AM

### Accessibility mode: 
Screen Shot 2023-09-07 at 11 26 01 AM

## Changes Made:

- 508 fixes: added Doctype and lang attribute, added alt text to image, removed style type attribute
- Internationalized text
- Added Accessibility mode functionality. Removed extra styles

## Pull Request Checklist:

- [X] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [N/A] Hotfixes should be merged into both the `Development` and `master` branches (at the same time).
- [X] All new text is preferrably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages)
- [X] There are no linter errors
- [ ] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [X] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [X] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [X] Comment which GitHub issue(s), if any does this PR address
- [X] It is the code author's responsibility to merge their own pull request after it has been approved
- [X] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [N/A] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [N/A] If the dev team has agreed that this PR represents the last PR going into the Development branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place

Thanks for contributing and keeping it clean!

### CODE REVIEW

**Comment by themerekat** [comment]
(I know I'm not slated as a reviewer, but this is looking great! Thanks @NikitaSalikov !!)

File: ident/admin/index.php [hunk 1]
```diff
@@ -111,15 +114,17 @@ function openHeadingAdmin(){
 										}
 										?>
 									</select>
-									<a href="#" onclick="openHeadingAdmin(); return false;"><img src="../../images/edit.png" /></a>
+									<a href="#" onclick="openHeadingAdmin(); return false;"> <img src="../../images/edit.png" alt="Create new group" /></a>
```
Comment (Atticus29) [inline ident/admin/index.php]: Internationalize

**Review by Atticus29 — COMMENTED** [review]
Very few changes; nice work (see comments in the code).

TVP says:
Screen Shot 2023-09-12 at 2 34 08 PM

I think that the solution to this is to add the `autocomplete: off` code inline in the `<input>` elements:

`<input type="text" name="username" autocomplete="off">`

**Review by Atticus29 — APPROVED** [review]
LGTM!

**Pull request merged**